// util/data.js
  // Sample data for mockup testing purposes

  export const bloques = [
    {
      name: 'Directivos',
      roles: `
        <p>Presidente/a</p>
        <p>Vicepresidente/a</p>
        <p>Tesorero/a</p>
        <p>Secretario/a</p>
      `,
    },
    {
      name: 'Órgano de Control',
      roles: `
        <p>Fiscal</p>
      `,
    },
    {
      name: 'Conciliación<br> y Vigilancia',
      roles: `
        <p>Conciliador #1</p>
        <p>Conciliador #2</p>
        <p>Conciliador #3</p>
      `,
    },
    {
      name: 'Representación',
      roles: `
        <p>Delegado #1</p>
        <p>Delegado #2</p>
        <p>Delegado #3</p>
      `,
    },
    {
      name: 'Comisiones<br> de trabajo',
      roles: `
        <p>Coordinadores/as<br> de comisiones<br> del trabajo</p>
      `,
    },
  ];

  export const planchas = [
    { name: 'Claro que sí '},
    { name: 'Claro que sí '},
    { name: 'Claro que sí '},
    { name: 'Claro que sí '},
    { name: 'Claro que sí '},
    { name: 'Claro que sí '},
    { name: 'Claro que sí '},
    { name: 'Claro que sí '},
  ];

  export let team = [
    {avatar: 'gonzalez_gutierrez.jpg', name:'Maribel González Gutiérrez', role: 'Presidenta'},
    {avatar: 'caceres_garcia.jpg', name:'Silvia Cáceres García', role: 'Vicepresidenta'},
    {avatar: 'garcia_ballester.jpg', name:'María García Ballester', role: 'Tesorera'},
    {avatar: 'juarez_martinez.jpg', name:'Anabel Juárez Martínez', role: 'Secretaria'},
  ];

  export let items = [
    { 
      logo: `gid1.svg`,
      header: 'Consejo Radical',
      group: team,
      visible: false,
    },
    { 
      logo: `gid2.svg`,
      header: 'Claro que sí',
      group: team,
      visible: false,
    },
    { 
      logo: `gid3.svg`,
      header: 'Consejo Radical',
      group: team,
      visible: false,
    },
  ]
